src/components/job-overview/step-summary.js

```diff
@@ -117,14 +146,21 @@ const StepSummary = ({ appName, jobName, step }) => (
         {step.name}
       </Link>
       <div>{getDescription(step)}</div>
+      {step.scan && (
+        <div>
+          <ScanSummary scan={step.scan}></ScanSummary>
+        </div>
+      )}
```

Comment:
May or may not be intentional, but you said you would place it at the end of the row.

Reply:
Fixed